.github/workflows/update-viablestrict.yml

```diff
@@ -0,0 +1,45 @@
+name: Update viable/strict
+
+on:
+  schedule:
+    - cron: 17,47 * * * *
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  do_update_viablestrict:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: Checkout PyTorch
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
+
+      - name: Install Python Packages
+        run: |
+          pip3 install rockset==0.8.10
```

Comment:
One thing we might want to do in the future is consolidate these dependencies into flat files, perhaps in something like `.github/dependencies/update-viablestrict.txt` and then just use `pip install -r .github/dependencies/update-viablestrict.txt`.

That way we have a singular place to look if we need to update dependencies like this.

Not saying we have to do this now but something to think about in the future.